tests/test_client.py

```diff
@@ -110,3 +110,38 @@ def test_client_invalid_auth_token(mocker, mock_authorizer_tuple, token, mock_ke
     # Call the Garden constructor and expect an auth exception
     with pytest.raises(AuthException):
         GardenClient(auth_client=mock_auth_client)
+
+
+def test_local_storage_garden(mocker, garden_client, garden_all_fields, tmp_path):
+    # mock to replace "~/.garden/db"
+    mocker.patch("garden_ai.client.LOCAL_STORAGE", new=tmp_path)
+    uuid = garden_all_fields.uuid
+    garden_client.put_local(garden_all_fields)
+    record = garden_client.get_local(uuid)
```

Comment:
It seems like the existing application code is still writing to {garden_id}.json right now, right? Is it a big lift to convert the code that's persisting models now so these parts aren't out of sync?